documentation/text-matching-ranking.html

```diff
@@ -134,12 +135,77 @@ <h3 id="query-trace">Query Trace</h3>
 </p>
 
 
-<!-- ToDo: prefix/substring -->
-<!-- ToDo: streaming search note -->
-<!-- ToDo: refer to
-  https://docs.vespa.ai/documentation/reference/schema-reference.html#match
-  https://docs.vespa.ai/documentation/reference/schema-reference.html#indexing
-  and write some guidance -->
+<h3 id="prefix-search">Prefix Search</h3>
+<p>
+A prefix search will match the query term "hea" to documents with "head".
+However, prefix search is <span style="text-decoration: underline"> not</span>
+enabled in text matching using <em>index</em> mode - some alternatives:
+</p>
+<table class="table">
+  <tr>
+    <th style="white-space: nowrap">Array of string attributes</th>
+    <td>
+      Use an attribute with array of strings in addition to the index string field.
```

Comment:
Consider mention that prefix search is also supported for single string attributes.

Reply:
will fix